packages/core/src/utils/meta.ts

```diff
@@ -22,8 +25,15 @@ import { getTraceData } from './traceData';
  * ```
  *
  */
-export function getTraceMetaTags(span?: Span, scope?: Scope, client?: Client): string {
-  return Object.entries(getTraceData(span, scope, client))
-    .map(([key, value]) => `<meta name="${key}" content="${value}"/>`)
-    .join('\n');
+export function getTraceMetaTags({
+  span,
+  scope,
+  client,
+  asArray,
+}: { span?: Span; scope?: Scope; client?: Client; asArray?: boolean } = {}): string | string[] {
```

Comment:
_if_ we move forward with this, we can adjust the types so that TS knows that if `asArray` is true, it returns an array and otherwise a string

Reply:
The type checking should already check this as I created two function overloads above. But I am open for another approach (in the case this gets merged) :D

Reply:
sorry, I missed this. This is what I had in mind :)